src_python/habitat_sim/utils/settings.py

```diff
@@ -11,24 +11,22 @@
     "width": 640,
     "height": 480,
     "default_agent": 0,
-    "sensor_height": 1.5,
-    "hfov": 90,
-    "color_sensor": True,
-    "semantic_sensor": False,
-    "depth_sensor": False,
-    "ortho_rgba_sensor": False,
-    "ortho_depth_sensor": False,
-    "ortho_semantic_sensor": False,
-    "fisheye_rgba_sensor": False,
-    "fisheye_depth_sensor": False,
-    "fisheye_semantic_sensor": False,
-    "equirect_rgba_sensor": False,
-    "equirect_depth_sensor": False,
-    "equirect_semantic_sensor": False,
     "seed": 1,
     "physics_config_file": "data/default.physics_config.json",
+    "sensors": [
```

Comment:
Instead of a list of dicts, maybe we use a dict of dicts? The key could be the uuid which would prevent duplication and allow dict merging overrides.